templates/post-single-author.adoc

```diff
@@ -15,6 +16,9 @@ AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
 // Above:
 // Do not insert any blank lines between any of the lines above.
 //
+//
+// "open-graph-image" is set to OL logo, howver it can be changed to a different image or left blank in which case it will be set to the default  // Open Liberty logo
```

Comment:
add comment that whenever possible, a more appropriate/specific image should be used.  For example, if present, to an image used within the blog post.

Reply:
also, seems like formatting of comments is off

Reply:
Done